CONTRIBUTING.md

```diff
@@ -96,3 +98,10 @@ Lowercase is preferred for file and folder names, one exception to this is Files
 - `src/components` will contain reusable UI component for eg. `Button`, `Input` etc.
 - If a style is specific only to the component, and cannot be used anywhere else, add it to `<ComponentName>.styles.ts` in the same folder as the component.
 - All css properties that require number as input should be in multiple of 4 and should be used from `src/common/constants/cssScale.ts`.
+
+### Testing
+
+- Write test cases for each component in their directory for e.g. `src/Home/Home.test.tsx` you can also refer to directory structure.
+- Make sure you cover at least 70% test-case scenario.
+- You can generate "test coverage" for modified files using `yarn test:coverage` command.
```

Comment:
Wouldn't this command generate coverage for the whole project?

If we are running it locally we want to on generate coverage for the new changes.

Reply:
It generate coverage for new changes

Reply:
Can point me to resource for the same. All I was able to find was this: https://create-react-app.dev/docs/running-tests/#coverage-reporting

From this doc it seems like it calculates coverage for the whole project

Reply:
There is no resource for it but I think it is handled via CRA script. I also couldn't find any relevant resources as such but I ran the command and it check for modified files only.

Reply:
Let's remove this from our document for now then, we can add it later after confirming when we have multiple tests.

Reply:
I confirmed it myself by writing two test files